thrift0.13/src/main/java/com/linecorp/armeria/common/thrift/ThriftProtocolFactories.java

```diff
@@ -27,7 +27,7 @@
 import com.linecorp.armeria.common.thrift.text.TTextProtocolFactory;
 
 /**
- * Provides a set of the known {@link TProtocolFactory} instances.
+ * Holds a few known {@link TProtocolFactory} instances.
```

Comment:
Holds -> Provides ?

Reply:
Umm, why? This class doesn't provide anything

Reply:
Hmm, doesn't it provide `TProtocolFactory` instances as static fields?

Reply:
Let me change this to move forward for today's release.